### ISSUE TITLE
Add ofIQueryable method

### ISSUE THREAD

**Issue by xperiandri** [fsprojects/FSharp.Control.AsyncSeq#111]
### Actual behavior

Currently to produce `AsynSeq` from `IQueryable` I have to write
``` F#
(query {
    for biblioteko in context.Bibliotekoj do
        where (biblioteko.ID = bibliotekoId)
        select biblioteko
 }).AsAsyncEnumerable()
|> AsyncSeq.ofAsyncEnum
|> AsyncSeq.tryFirst
```

### Expected behavior

But I could write
``` F#
(query {
    for biblioteko in context.Bibliotekoj do
        where (biblioteko.ID = bibliotekoId)
        select biblioteko
 })
|> AsyncSeq.ofIQueryable
|> AsyncSeq.tryFirst
```

### Related information 

* FSharp.Control.AsyncSeq 2.0.23

**Comment by dsyme** [comment]
Can you send a PR for this?

**Comment by xperiandri** [comment]
Yes, I will

**Comment by xperiandri** [comment]
@dsyme done in https://github.com/fsprojects/FSharp.Control.AsyncSeq/pull/112

**Comment by wilsoncg** [comment]
Released in 2.0.24
